packages/block-library/src/missing/edit.native.js

```diff
@@ -25,7 +25,20 @@ export default class UnsupportedBlockEdit extends Component {
 		const icon = blockType ? normalizeIconObject( blockType.settings.icon ) : 'admin-plugins';
 
 		return (
-			<View style={ styles.unsupportedBlock }>
+			<View style={ styles.unsupportedBlock }
+				accessible={ true }
+				accessibilityLabel={
+					blockType ?
+						sprintf(
+							/* translators: accessibility text. %s: unsupported block type. */
+							__( 'Unsupported block: %s.' ),
```

Comment:
The dots at the end are not necessary.
I just checked and the Heading block also has a dot at the end when it's empty, but I think we should correct that one too.

```suggestion
							__( 'Unsupported block: %s' ),
```